src/main/java/com/fasterxml/jackson/core/JsonParser.java

```diff
@@ -1013,6 +1013,15 @@ public void finishToken() throws IOException {
      * Method can be called for any token type.
      */
     public abstract String getText() throws IOException;
+    
+    /**
+     * Method to read the textual representation of the current token in chunks and 
+     * pass it to the given Writer
+     * 
+     *  @return true, if the text chunks has been processed completely and passed to the given writer
+     *          false, if the text chunks are still being processed
+     */
+    public abstract boolean readText(Writer writer) throws IOException;
```

Comment:
Please add `@since 2.8` here to indicate when method was introduced.

Also, for backwards compatibility, please make this throw `UnsupportedOperationException`, so that existing non-json implementations will not fail to load with 2.8.